ROMFS/px4fmu_common/init.d/rc.interface

```diff
@@ -23,6 +23,20 @@ then
 	set MIXER_AUX none
 fi
 
+#
+# If mount (gimbal) control is enabled and output mode is AUX, set the aux
```

Comment:
This should be above `if ver hwcmp AEROFC_V1 AV_X_V1 CRAZYFLIE MINDPX_V2 NXPHLITE_V3 PX4FMU_V4 OMNIBUS_F4SD`, since that takes priority.

Reply:
Done. Thanks!